src/mobile/src/ui/components/SeedVaultExportComponent.js

```diff
@@ -20,6 +20,7 @@ import InfoBox from './InfoBox';
 import Button from './Button';
 import CustomTextInput from './CustomTextInput';
 import PasswordFields from './PasswordFields';
+import { removeNonAlphaNumeric } from '../../../../shared/libs/utils';
```

Comment:
```suggestion
import { removeNonAlphaNumeric } from 'shared-modules/libs/utils';
```
We recently added https://github.com/tleunen/babel-plugin-module-resolver so now the `../..` is no longer needed. If you want you can change this, but it's not a huge issue.